content/hi/docs/concepts/architecture/leases.md

```diff
@@ -0,0 +1,102 @@
+बाकी प्रणाली। जबकि अपने आप में विशेष रूप से उपयोगी नहीं है, यह ग्राहकों के लिए एक तंत्र प्रदान करता है
+पता लगाएँ कि `क्यूब-एपीसर्वर` के कितने उदाहरण कुबेरनेट्स नियंत्रण विमान का संचालन कर रहे हैं।
+क्यूब-एपिसर्वर लीज़ का अस्तित्व भविष्य की क्षमताओं को सक्षम बनाता है जिनके बीच समन्वय की आवश्यकता हो सकती है
+प्रत्येक क्यूब-एपीसर्वर।
+
+आप `क्यूब-सिस्टम` नेमस्पेस में लीज ऑब्जेक्ट्स की जांच करके प्रत्येक क्यूब-एपीसर्वर के स्वामित्व वाले पट्टों का निरीक्षण कर सकते हैं
+`क्यूब-एपीसर्वर-<sha256-हैश>` नाम से। वैकल्पिक रूप से आप लेबल चयनकर्ता `k8s.io/component=kube-apiserver` का उपयोग कर सकते हैं:
+
+```shell
+kubectl -n kube-system get lease -l k8s.io/component=kube-apiserver
+```
+```
+NAME                                        HOLDER                                                                           AGE
+kube-apiserver-c4vwjftbvpc5os2vvzle4qg27a   kube-apiserver-c4vwjftbvpc5os2vvzle4qg27a_9cbf54e5-1136-44bd-8f9a-1dcd15c346b4   5m33s
+kube-apiserver-dz2dqprdpsgnm756t5rnov7yka   kube-apiserver-dz2dqprdpsgnm756t5rnov7yka_84f2a85d-37c1-4b14-b6b9-603e62e4896f   4m23s
+kube-apiserver-fyloo45sdenffw2ugwaz3likua   kube-apiserver-fyloo45sdenffw2ugwaz3likua_c5ffa286-8a9a-45d4-91e7-61118ed58d2e   4m43s
+```
+
+पट्टे के नाम में प्रयुक्त SHA256 हैश OS होस्टनाम पर आधारित है जैसा कि उस API सर्वर द्वारा देखा गया है। प्रत्येक क्यूब-एपीसर्वर होना चाहिए
+एक होस्टनाम का उपयोग करने के लिए कॉन्फ़िगर किया गया है जो क्लस्टर के भीतर अद्वितीय है। क्यूब-एपीसर्वर के नए उदाहरण जो एक ही होस्टनाम का उपयोग करते हैं
+नई लीज़ वस्तुओं को तत्काल करने के विपरीत, एक नई धारक पहचान का उपयोग करके मौजूदा पट्टों का अधिग्रहण करेगा। आप चेक कर सकते हैं
+क्यूब-एपिसेवर द्वारा उपयोग किया जाने वाला होस्टनाम `kubernetes.io/hostname` लेबल के मान की जाँच करके:
+
+```shell
+kubectl -n kube-system get lease kube-apiserver-c4vwjftbvpc5os2vvzle4qg27a -o yaml
+```
+```yaml
+apiVersion: coordination.k8s.io/v1
+kind: Lease
+metadata:
+  creationTimestamp: "2022-11-30T15:37:15Z"
+  labels:
+    k8s.io/component: kube-apiserver
+    kubernetes.io/hostname: kind-control-plane
+  name: kube-apiserver-c4vwjftbvpc5os2vvzle4qg27a
+  namespace: kube-system
+  resourceVersion: "18171"
+  uid: d6c68901-4ec5-4385-b1ef-2d783738da6c
+spec:
+  holderIdentity: kube-apiserver-c4vwjftbvpc5os2vvzle4qg27a_9cbf54e5-1136-44bd-8f9a-1dcd15c346b4
+  leaseDurationSeconds: 3600
+  renewTime: "2022-11-30T18:04:27.912073Z"
+```
+
+क्यूब-एपिसर्वर के समाप्त हो चुके पट्टे जो अब मौजूद नहीं हैं, नए क्यूब-एपिसर्वर द्वारा 1 घंटे के बाद कचरा एकत्र किए जाते हैं।
+
+आप `APIServerIdentity` को अक्षम करके API सर्वर पहचान पट्टे को अक्षम कर सकते हैं
+[फीचर गेट](/docs/reference/command-line-tools-reference/feature-gates/)।
+
+## वर्कलोड {#कस्टम-वर्कलोड}
+
+आपका अपना कार्यभार पट्टों के अपने स्वयं के उपयोग को परिभाषित कर सकता है। उदाहरण के लिए, आप एक कस्टम चला सकते हैं
+{{< glosary_tooltip term_id="controller" text="controller" >}} जहां एक प्राथमिक या नेता सदस्य
```

Comment:
Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.